#Datasource
#database=postgres
spring.datasource.url=jdbc:postgresql://google/postgres?useSSL=false&socketFactoryArg=ciscochatbot-4eed6:asia-northeast1:la&socketFactory=com.google.cloud.sql.postgres.SocketFactory&user=postgres&password=la123456
spring.datasource.driverClassName=org.postgresql.Driver

spring.cloud.gcp.sql.database-name=la
spring.cloud.gcp.sql.instance-connection-name=ciscochatbot-4eed6:asia-northeast1:la
spring.datasource.username=postgres
spring.datasource.password=la123456
# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true

#spring.cloud.gcp.project-id=
#spring.cloud.gcp.credentials.location=file:/path/to/service-account.json

#Bot username: LnDBot@sparkbot.io
boltToken=YjI0NDYxNjktYjFjYS00NTUyLTllZjQtMDU4OWIwMDgyZDdlNzAwNGM5NzUtZTgz